examples/ReactExample/package.json

```diff
@@ -7,20 +7,23 @@
     "test": "jest"
   },
   "dependencies": {
-    "react": "16.0.0",
-    "react-native": "0.50.4",
-    "react-navigation": "^1.0.0-beta.21",
-    "react-native-exit-app-no-history": "^1.0.2",
-    "realm": "file:../.."
+    "react": "16.6.3",
+    "react-native": "0.57.8",
+    "react-native-exception-handler": "^2.4.3",
+    "react-native-fs": "^2.8.5",
+    "react-navigation": "^1.0.0",
+    "xmlbuilder": "^4.2.1"
+  },
+  "resolutions": {
+    "moment": "2.19.1"
   },
   "devDependencies": {
-    "babel-jest": "18.0.0",
-    "babel-preset-react-native": "4.0.0",
+    "eslint": "^3.0.0",
+    "install-local": "^0.6.2",
```

Comment:
Looks like 1.0.0 shipped 8 days ago.

As this is an example I feel it's a little misleading to install using this package. People might be looking at it as a reference on how to install Realm JS and get confused. The integration test I just merged had the same issue and I solved it by packing Realm JS and installing it from the package install script like so: https://github.com/realm/realm-js/blob/master/integration-tests/environments/react-native/package.json#L10 .. I don't know if that will be less confusing - but I wanted to mention the alternative

Reply:
Manually acquiring a tarball and calling npm install on that is significantly more dissimilar to how end users should install realm than this is. The point of using install-local is that it makes it almost identical to a normal dependency other than that it's in a separate section in package.json.

Reply:
There is also a case to be made that this will actually not be using the .tgz that will eventually ship to users and instead it will package its own. Installing from the .tgz will catch more errors from potential issues with the way we might choose to package the archive.

Reply:
install-local just calls `npm pack` and installs from that.

Reply:
The reason we’ve had issues with regressions not being caught by tests in the past isn’t due to minor details around how realm is installed in tests: it’s because the node release is built from realm-js-private with a different build system and some important code differences, and while rjp has stuff set up to run the public tests against that module, it’s been broken for years. Assembling a tarball from this repo and installing from that isn’t any closer to testing the thing that users will actually install than anything else these tests have been doing.

Reply:
Building from the package that we've already build at this point might speed up installing this app. I don't have a strong opinion here, thus it's not a blocker from my perspective - just wanted to share my thoughts.